cmake/compiler/arcmwdt/compiler_flags.cmake

```diff
@@ -187,3 +186,6 @@ set_compiler_property(PROPERTY no_position_independent
 
 # Required ASM flags when using mwdt
 set_property(TARGET asm PROPERTY required "-Hasmcpp")
+
+# Pass includes directly into assembly build
+set_property(TARGET asm PROPERTY required "-I${TOOLCHAIN_HOME}/arc/inc")
```

Comment:
I'm wondering if we have duplication of `${TOOLCHAIN_HOME}/arc/inc` include path for ASM targets in case of build with MINIMAL LIBC?

Reply:
Probably no.
`-I${TOOLCHAIN_HOME}/arc/inc` passed to ccac just once.
It seems to be filtered as duplication on some stage.

Reply:
I've checked build - we pass the same flag twice for ASM build. I.e.: for fast_irq.S build (nsim_hs platform)

```
-I/home/paltsev/CodeData/debug/metaware/metaware_2022_06_RC1/MetaWare/../MetaWare/arc/inc
[snip]
-I/home/paltsev/CodeData/debug/metaware/metaware_2022_06_RC1/MetaWare/../MetaWare/arc/inc
```